flatpak_builder_lint/staticfiles/exceptions.json

```diff
@@ -3531,6 +3531,8 @@
     },
     "io.github.ungoogled_software.ungoogled_chromium": {
         "finish-args-contains-both-x11-and-wayland": "Predates the linter rule",
+	"finish-args-dconf-talk-name": "Access host dconf for proxy resolution",
+        "finish-args-direct-dconf-path": "Access host dconf for proxy resolution",
         "manifest-has-bundled-extension": "Predates the linter rule"
```

Comment:
can you fix the indentation?

![image](https://github.com/user-attachments/assets/5591d4ba-6b22-4b6e-8bb1-bf3d24ec01c0)

Reply:
Sorry, I used the GitHub web editor. It looked OK when I used that :)